chainerio/filesystems/hdfs.py

```diff
@@ -16,17 +18,51 @@
 logger.addHandler(logging.StreamHandler())
 
 
+def parse_klist_output(output):
```

Comment:
As this is not to be exposed to users, it would be nice if the name starts with `_`